lib/propolis/src/hw/nvme/admin.rs

```diff
@@ -258,13 +258,19 @@ impl NvmeCtrl {
                 // `ncqa`/`nsqa` are 0-based values so subtract 1
                 cmds::Completion::success_val((ncqa - 1) << 16 | (nsqa - 1))
             }
+            // NVMe 1.0e Figure 66 Identify - Identify Controller Data Structure
+            // "If a volatile write cache is present, then the host may ...
+            //  control whether it is enabled with Set Features specifying the
+            //  Volatile Write Cache feature identifier."
```

Comment:
style nit: perhaps move the comment into the match case body, to match what you did for `AsyncEventReq`?